src/components/common/AppBar/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, {useMemo} from 'react';
+import {NavLink} from 'react-router-dom';
+
+import {AppBar as NavBar, Box, Toolbar, IconButton, Typography, Menu, MenuItem, Container} from '@mui/material';
+import CodeOffIcon from '@mui/icons-material/CodeOff';
+import MenuIcon from '@mui/icons-material/Menu';
+
+import {routes} from '../../../router/routes';
+
+const pages = [{
+    title: 'Courses',
+    link: routes.courseList
+}];
+
+export const AppBar = () => {
+    const navElements = useMemo(() => pages.map((page) => (
+        <MenuItem key={page.title}>
+            <NavLink to={page.link}>{page.title}</NavLink>
+        </MenuItem>
+    )), []);
```

Comment:
Add dependencies, which would cause update.
Otherwise, it will behave as static variable.

Reply:
Actually, on this stage of application, i don't need navigation menu to be updated)